src/common/utility/maptypes.h

```diff
@@ -4,14 +4,39 @@
 
 #ifndef VISIT_MAP_TYPES_H
 #define VISIT_MAP_TYPES_H
+#include <algorithm>
+#include <map>
+#include <set>
 #include <string>
 #include <vectortypes.h>
-#include <map>
 
 typedef std::map<std::string, int>          StringIntMap;
 typedef std::map<std::string, std::string>  StringStringMap;
 typedef StringIntMap                        LevelColorMap;
 typedef std::map<std::string, stringVector> StringStringVectorMap;
 typedef std::map<double, int>               DoubleIntMap;
 
+
+// CI short for Case-Insenstive
+struct CIComparator {
+  bool operator() (const std::string& s1, const std::string& s2) const
```

Comment:
FYI...I tried an imperical experiment on my OSX system changing the body of this method to just this one line...

```
      return strcasecmp(s1.c_str(), s2.c_str()) < 0;
```

And then used `sesame.pdb` (which has a ton of variables in it so any performance issues should manifest in an observable way). I also set `VISIT_FORCE_SPECULATIVE_EXPRESSION_GENERATION` to ensure the variable lists were even longer. I then manually timed the two implementations from hitting `return` on the terminal to seeing un-grayed menus in the GUI. I did **NOT** notice as much of a performance variance as I expected. I did see some, maybe 4 seconds in 40 or about 10%. I think if the variable names were longer, it would be worse. Given that, its hard to make a strong argument to request a change here. OTOH, the coding is also much simpler. And, FYI, a few of the *nice* C++-esque ways of doing this well involve introduce a new string type and are therefore horribly inconvenient.

Reply:
I suspect that in the current GUI implementation, there is enough per-variable Qt overhead that efficiencies in string comparisons of variable names are not the dominant factor. If we improve Qt overheads though...this situation may change.